frontend/src/components/Hacknight.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-container fluid text-xs-center>
+    <v-row align="center" justify="center">
+      <v-col class="d-flex" cols="12" sm="6">
+        <v-alert
+          type="error"
+          :value="getError ? true : false"
+          transition="slide-y-transition"
+          dismissible
+          >{{ getError }}</v-alert
+        >
+      </v-col>
+    </v-row>
+    <v-row align="center" justify="center">
+      <v-col class="d-flex" cols="12" sm="6">
+        <v-select
+          v-model="selectedHacknight"
+          :items="getHacknights"
+          item-text="date"
+          item-value="id"
+          label="Select Hacknight"
+          @input="onGetHacknight"
+        >
+          <template v-slot:append-outer>
+            <v-btn
```

Comment:
To avoid in-line styling, instead of current <v-btn/> i would use:

`<v-btn`
`  text`
`  class="Y"`
`  @click="onCreateHacknight"`
`>`

and in the <style> section:

`.Y {`
`  align-content: center;`
`  background-color: $blue;`
`  color: $white;`
`  display: flex;`
`  justify-content: center;`
`  margin: 4px;`
`  top: -12px;`
`}`

It assumes we get rif off things that seem do nothing :)